hack/Dockerfile

```diff
@@ -0,0 +1,33 @@
+#
```

Comment:
What about putting a separate Dockerfile into the respective directory.

I.e. ls images/libvirt-kubevirt/Dockerfile.build

And here you could actually do a change where the -dev docker image, inherits the runtime docker image. That way you have the runtime rpm installation part in the runtime image only.

Reply:
I'm looking into this and the way it is structured now you would get a circular dependency as the docker build depends on the binaries to be built. We could, however, create the libvirt-kubevirt container before building, then create the -dev one, then build it and then continue with other containers. I'll look into that.